README.md

```diff
@@ -89,7 +89,7 @@ ON CKAN <= 2.8:
 
 Once installed, add the `validation` plugin to the `ckan.plugins` configuration option in your INI file:
 
-    ckan.plugins = ... validation
+    ckan.plugins = ... validation validation_resource validation_package
```

Comment:
need more description the the separation that has occurred here. Before you just needed to add ```validation``` now you need ```validation validation_resource validation_package``` so what does _resource and _package add that the original now does not do?

Reply:
@mutantsan , can you point me at the documentation you read to come to that idea. 
Or 
Add more details to the readme (or this comment chain) on why validation now requires 3 entry points to be added when it was just one prior.

Reply:
Mentioned interfaces have similar method names (after_create, after_update), so the previous implementation is a mess in terms of distinguishing the method calls. You are free to combine them, if you think that this will improve something, except for saving another portion of bytes in config.